docs/install/from_source.rst

```diff
@@ -174,12 +174,12 @@ Method 1
 
    Set the environment variable `PYTHONPATH` to tell python where to find
    the library. For example, assume we cloned `tvm` on the home directory
-   `~`. then we can added the following line in `~/.bashrc`.
+   `~` then we can add the following line in `~/.bashrc`.
    The changes will be immediately reflected once you pull the code and rebuild the project (no need to call ``setup`` again)
 
    .. code:: bash
 
-       export TVM_HOME=/path/to/tvm
+       export TVM_HOME=~/tvm
```

Comment:
This may cause confusion since it's not necessary to put tvm under `$HOME`.

Reply:
I agree, but if you read the text preceding this statement it says
```For example, assume we cloned tvm on the home directory ~. then we can added the following line in ~/.bashrc```
Pasting the example code into your .bashrc file will not work since the /path/to/tvm is incorrect in this case. We can either remove the "For example..." text and leave `/path/to/tvm` or change the path to ~/tvm.
As it stands it is not correct

Reply:
Ah that's a good point. I'll prefer to keep `/path/to/tvm` and change to "For example, assume we cloned `tvm` to `/path/to/tvm` then..."

Reply:
done